client/src/cmdhfemrtd.c

```diff
@@ -1296,6 +1291,15 @@ int infoHF_EMRTD(char *documentnumber, char *dob, char *expiry, bool BAC_availab
     return PM3_SUCCESS;
 }
 
+static void text_to_upper(uint8_t *data, int datalen) {
+    // Loop over text to make lowercase text uppercase
+    for (int i = 0; i < datalen; i++) {
+        if ('a' <= data[i] && data[i] <= 'z') {
+            data[i] -= 32;
+        }
+    }
+}
+
```

Comment:
there is a  strup function to use...

Reply:
I checked the code and couldn't find something like that, where is it?

Reply:
toupper